src/Gaffer/TweakPlug.cpp

```diff
@@ -112,17 +112,39 @@ vector<T> tweakedList( const std::vector<T> &source, const std::vector<T> &tweak
 {
 	vector<T> result = source;
 
+	struct HashFunc
+	{
+		size_t operator()(const T& x) const
+		{
+			IECore::MurmurHash h;
```

Comment:
I hadn't considered the need for a hash function for every type we might tweak - this seems like a neat solution.